howto/instance/create.md

```diff
@@ -71,6 +71,22 @@ Launch a raw instance by providing the image ID in the following command:
 
 See [Provided images](https://discourse.ubuntu.com/t/provided-images/24185) for a list of images that are available in Anbox Cloud.
 
+### Launch an instance with streaming enabled
+
+*since 1.23.0*
+
+If you want to stream the visual output from an instance, specify it at the time of instance creation. Otherwise, the instance cannot be streamed. In previous versions of Anbox Cloud, it was only possible to create an instance that has streaming enabled via the [Stream Gateway API](https://canonical.github.io/anbox-cloud.github.com/latest/anbox-stream-gateway/). The same can now be achieved through the [AMS API](https://canonical.github.io/anbox-cloud.github.com/latest/ams/) or the `amc` CLI.
+
```

Comment:
Is it worth mentioning how the agent API token is created? Another question is who will be in charge of creating the agent API token? charm or end user?

Reply:
It will all be automatically setup for the user in all cases (microbox, appliance or charms)

Reply:
Alright, that's the missing piece for now. 
We should create a story to track it.

Reply:
https://warthogs.atlassian.net/browse/AC-2415